src/models/city.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (sequelize, DataTypes) => {
+  const City = sequelize.define("Cities", {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    gallary: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    location: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    countryId: {

```

Comment:
this **countryId** field should be the first field in this model structure and not last.